templates/neutron_ovn_metadata_agent.ini

```diff
@@ -0,0 +1,11 @@
+# NOTE: some necessary metadata agent options are set by nova.
+# Specifically, nova_metadata_host and metadata_proxy_shared_secret.
+#
+# The final metadata agent deployment will have to merge settings from multiple
+# sources.
+
+[DEFAULT]
+transport_url = {{ .TransportURL }}
```

Comment:
Do we really need this in neutron-ovn-metadata-agent? I don't think this agent is connecting to the rabbitmq at all.

Reply:
I didn't realize it doesn't use RPC for any maintenance tasks like agent healthcheck reporting. I will remove this.